src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,21 @@
+$color-blue-accent: #00acdc;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-white: #fff;
```

Comment:
it's redundant cause white can't be changed

Reply:
@generatedman Yes, sounds weird, like white is always white and so does black. But I mean that there are lots of shades of white, and In that particular project white is #fff and not #fefcf3. Maybe I should name it color-light or something.